pkg/models/tickets.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+type TicketCreation struct {
+	Enabled bool `yaml:"enabled,omitempty"`
+	Jira    Jira `yaml:"jira,omitempty"`
```

Comment:
i think this makes more sense since we default to github issues.
theres no need for an enabled field, just specify jira if you want it to start being created there.
```suggestion
	Jira *Jira `yaml:"jira,omitempty"`
```

Reply:
we don't open separate github issues for every finding (above the severity threshold). it's only just the dashboards. the behaviour atm is if Jira is not installed/specified, a github issue will be created for each finding (on top of the dashboards). Since it just loops through newFindings and calls CreateTicket(finding).

Also what happens if they put Jira in the global config file but want to disable ticket creation on certain repos (e.g. don't create tickets for my internal tool python script that is only run locally).

Reply:
integrations:
  jira:
    disabled: true
    project_key:
    issue_type:
    threshold: